src/components/dialog-full-screen/dialog-full-screen.test.js

```diff
@@ -278,56 +280,71 @@ context("Testing DialogFullScreen component", () => {
 
   describe("should check accessibility for Dialog Full Screen", () => {
     it("should check accessibility for default Dialog Full Screen component", () => {
-      CypressMountWithProviders(<DialogFullScreenComponent open />);
+      CypressMountWithProviders(<DefaultDocsStory />);
```

Comment:
this makes no sense to test closed DialogFullScreen

Reply:
I've added the action on line 285 in order to handle this - so the button is clicked to open the dialog then the check is carried out :)